docs/source/RunningInEmulationTCG.rst

```diff
@@ -0,0 +1,92 @@
+.. _emulation:
+
+===============================
+Running in emulation mode (TCG)
+===============================
+
+Sometimes it's useful to run Avocado-vt in emulation mode (eg. when
+checking PR about different architecture, or to debug certain feature
+while executing the test). As this is not default, extra arguments are
+usually required.
+
+Shared for all architectures is that you need the image. We supply
+JeOS for most of the architectures, so you should be able to get
+it via::
+
+    $ avocado vt-bootstrap --vt-guest-os JeOS.$VERSION.$ARCH
+
+where:
+
+* ``VERSION`` is JeOS version (when writing this document it was ``27``)
+* ``ARCH`` is the desired architecture (eg. aarch64)
+
+Or you can run any of the ``unattended_install`` tests with
+``--vt-guest-os`` of your choice (very slow).
+
+When running the tests, on top of the usual arguments, you usually
+need to specify:
+
+* ``--vt-qemu-bin`` - path to binary that supports expected architecture
+  (eg. ``/usr/local/bin/qemu-system-aarch64``)
+* ``--vt-qemu-dst-bin`` - for migration tests you need also to specify
+  the destination qemu binary (otherwise default one is used)
+* ``enable_kvm=no`` - to disable KVM (if necessary)
+
+
+aarch64
+=======
+
+ARM always requires `cpu_model` as well as `machine_type`. To
+get list of available models you can run ``qemu-system-aarch64
+-cpu help -M virt`` (note: not all listed cpus are bootable).
+ARM also requires GIC version to be specified. Avocado-vt
+sets automatically the best via ``gic=host``, but that is not
```

Comment:
The option name is 'gic-version', not 'gic'. It has a default value (2) in virt machine. So it doesn't have to be specified. I think the reason you suggest to override 'machine_type_extra_params' parameter is to unset the default value in avocado-vt, right? How about remove line 42 (it's a bit inaccurate)?

Reply:
Sure, I'll update the `gic-version` and rephrase the note by explicitly mentioning the Avocado-vt default...

Reply:
LGTM.